### ISSUE TITLE
Outdated scheduling in "Design Principles"

### ISSUE THREAD

**Issue by khalilbenhamouda** [investorveamly/react-tutorial#1]

**Comment by khalilbenhamouda** [comment]
This is a subtle distinction but a powerful one. Since you don’t call that component function but let React call it, it means React has the power to delay calling it if necessary. In its current implementation React walks the tree recursively and calls render functions of the whole updated tree during a single tick. However in the future it might start delaying some updates to avoid dropping frames.

If something is offscreen, we can delay any logic related to it. ... To be clear, we are not taking advantage of this right now.

As far as I understand, React is taking advantage of delaying updates? 
If that's the case, seems like this part of the docs is outdated.

**Comment by investorveamly** [comment]
You are totally right @khalilbenhamouda , thank you for pointing this out! I will make sure to remove the docs.

**Comment by khalilbenhamouda** [comment]
Thanks, but we need to work on the updated documentation.
